ansible/roles/csm.rr.kyverno_policy/files/apply_topology_policy_restart.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+#
+# MIT License
+#
+# (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+if ! kubectl apply -f $1
+then
+  echo "Unable to apply kyverno topology policy"
+  exit 1
+fi
+
+deployments_to_restart=("cray-hmnfd" "cray-hbtd" "hpe-slingshot-jackaloped" "hpe-slingshot-ogopogod" "slingshot-fabric-manager"
+                        "cray-dns-powerdns" "cray-powerdns-manager" "cray-dns-powerdns-manager" "cray-dhcp-kea" "cray-sls"
+                        "cray-smd" "cray-bss" "cray-tftp" "cray-ipxe-aarch64" "cray-ipxe-x86-64" "cray-sts" "cfs-ara"
+                        "cfs-hwsync-agent" "cfs-trust" "cray-cfs-api" "cray-cfs-api-db" "cray-cfs-batcher" "cray-cfs-operator"
+                        "cray-bos" "cray-bos-db" "gitea-vcs" "cray-bss" "cray-sls" "cray-power-control" "cray-capmc"
+                        "cray-tftp" "gitea-vcs")
+for service in "${deployments_to_restart[@]}"
+do
+  kubectl rollout restart deployment $service -n services
+  if [ $? -ne 0 ];
+  then
+    echo "Unable to restart deployment $service in services namespace"
```

Comment:
We don't want the script to fail in this case? How will the user know it failed, since if the script passes, the Ansible play will also pass?

Reply:
We cannot finish the review of the PR with this comment unresolved.